WTWSMS/localisation/WtWSMS_events.csv

```diff
@@ -1678,6 +1678,11 @@ hist.22.a;Interesting developments...;Des d
 hist.23.name;Domuit Vascones;Domuit Vascones;;;;;;;;;;;;x
 hist.23.desc;For many years Visigothic kings have claimed the subjugation of the savage Vascones, but they had risen again. This time they will not!;Pendant de nombreuses ann�es, les rois wisigoths ont revendiqu� l'assujettissement des vascons sauvages, mais ils s'�taient de nouveau soulev�s. Cette fois ils ne le feront pas !;;;;;;;;;;;;x
 hist.23.a;Glory to the Visigoths!;Gloire aux Wisigoths !;;;;;;;;;;;;x
+hist.24.name;The Antioch earthquake in 526;;;;;;;;;;;;;x
+hist.24.desc;A powerful earthquake has struck the ancient city of Antioch. It has caused severe damage to many of the buildings. Most of the damage however, is a result of the fires that went on for many days in the immediate aftermath of the earthquake, made worse by the wind. Estimates of the death toll for this earthquake vary between 250,000 and 300,000.;;;;;;;;;;;;;x
```

Comment:
I don't think rulers of the period had access to such precise death toll numbers, so I think it would be better to speak of "large casualties" or something of the nature.

Reply:
Done.